desktop-widgets/simplewidgets.cpp

```diff
@@ -490,6 +491,50 @@ void DiveComponentSelection::buttonClicked(QAbstractButton *button)
 		COMPONENT_FROM_UI(cylinders);
 		COMPONENT_FROM_UI(weights);
 		selective_copy_dive(&displayed_dive, targetDive, *what, true);
+		QClipboard *clipboard = QApplication::clipboard();
```

Comment:
we might want to add a static text in the dialog that data is copied to the clipboard too.
otherwise right now it's unclear.